cross-doc-explainer.md

```diff
@@ -142,22 +142,51 @@ To achieve the same level of control for cross document navigations we propose t
 
 ### `pageconceal`
 
-Authors need some point in an outgoing document to customize or stop a transition. It's possible we may need a new event here but this is still under consideration, see whatwg/html#9702.
+Authors need some point in an outgoing document to customize or skip the cross-document transition.
 
-### `pagereveal`
+The current proposed event at whatwg/html#9702 is `pageconceal`. This event is fired when a
+document is about to be unloaded due to a navigation, and contains the origin-accessible activation info for the next document. If this a navigation that triggers a view-transition due to a `@view-transition` rule, the event's `viewTransition` property would be a `ViewTransition` object (otherwise it will be null).
```

Comment:
```suggestion
document is about to be unloaded due to a navigation, and contains the origin-accessible activation info for the next document. If this navigation triggers a view-transition due to a `@view-transition` rule, the event's `viewTransition` property would be a `ViewTransition` object (otherwise it will be null).
```